src/languages/css.js

```diff
@@ -49,8 +49,8 @@ export default function(hljs) {
       {
         className: 'selector-pseudo',
         variants: [
-          { begin: ':(' + css.PSEUDO_CLASSES.join('|') + ')' },
-          { begin: ':(:)?(' + css.PSEUDO_ELEMENTS.join('|') + ')' }
+          { begin: '(?<!\\\\):(' + css.PSEUDO_CLASSES.join('|') + ')'},
+          { begin: '(?<!\\\\):(:)?(' + css.PSEUDO_ELEMENTS.join('|') + ')'}
```

Comment:
These (look-behind) are problematic until we hit v12 - as this would be a breaking change to introduce to v11 since it would change which browsers v11 supports (Safari was very late to this party).

Reply:
fixed it in the new pr